### PR TITLE
Applications: Audio: Resolved wrapping calculation

### PR DESCRIPTION
**Pull request by koffes**
- OCT-3204
- Resolved a calculation not considering wrapping
- Minor changes to naming and comments

### CODE REVIEW

File: applications/nrf5340_audio/src/audio/audio_datapath.c [hunk 1]
```diff
@@ -161,6 +161,20 @@ static struct {
 	} pres_comp;
 } ctrl_blk;
 
+/**
+ * @brief	Get the current number of blocks in the output buffer.
+ */
+static int used_blocks_get(void)
```
Comment (alexsven) [inline applications/nrf5340_audio/src/audio/audio_datapath.c:167]: Nit: 'Used' sounds like they are spent

```suggestion
static int filled_blocks_get(void)
```

**Review by alexsven — APPROVED** [review]

**Review by gWacey — APPROVED** [review]

File: applications/nrf5340_audio/src/audio/audio_datapath.c (hunk 1 above)
Comment (koffes) [reply applications/nrf5340_audio/src/audio/audio_datapath.c:167]: Agreed, changed.

Comment (alexsven) [reply applications/nrf5340_audio/src/audio/audio_datapath.c:167]: Did you forget to add before committing and pushing? There are no changes in the last commit

**Review by alexsven — CHANGES_REQUESTED** [review]
missing changes

**Pull request merged**